app/src/main/java/eu/kanade/tachiyomi/ui/library/LibraryPresenter.kt

```diff
@@ -173,6 +173,9 @@ class LibraryPresenter(
         // SY -->
         val filterStarted = preferences.filterStarted().get()
         val filterLewd = preferences.filterLewd().get()
+        val inSources = sourceManager.getCatalogueSources().associate {
+            Pair(it.id.toInt(), preferences.filterSources(it.id.toInt()).get())
```

Comment:
I generally dont use the Pair constructor, use the `to` keyword like this
`it.id to preferences.filterSources(it.id).get()`
Also make sure to use Long's